plugins/resin/src/main/java/com/navercorp/pinpoint/plugin/resin/interceptor/WebAppInterceptor.java

```diff
@@ -1,38 +1,27 @@
 package com.navercorp.pinpoint.plugin.resin.interceptor;
 
-import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import com.caucho.config.ConfigException;
-import com.caucho.config.types.FileSetType;
-import com.caucho.config.types.PathPatternType;
 import com.caucho.loader.EnvironmentClassLoader;
-import com.caucho.loader.LibraryLoader;
-import com.caucho.loader.Loader;
 import com.caucho.server.webapp.WebApp;
-import com.caucho.vfs.FilePath;
-import com.caucho.vfs.JarPath;
-import com.caucho.vfs.Path;
 import com.navercorp.pinpoint.bootstrap.context.ServerMetaDataHolder;
 import com.navercorp.pinpoint.bootstrap.context.TraceContext;
 import com.navercorp.pinpoint.bootstrap.interceptor.AroundInterceptor;
 import com.navercorp.pinpoint.bootstrap.logging.PLogger;
 import com.navercorp.pinpoint.bootstrap.logging.PLoggerFactory;
-import com.navercorp.pinpoint.plugin.resin.JarEntry;
-import com.navercorp.pinpoint.plugin.resin.JarMap;
-
-
 
 /**
  * 
- * @author huangpengjie@fang.com 
+ * @author huangpengjie@fang.com
  *
  */
 public class WebAppInterceptor implements AroundInterceptor {
```

Comment:
Let's try to get rid of the compile-time dependency to `com.caucho.server.webapp.WebApp` as this class is not available in *3.0.9*.

You can do this by changing all `WebApp` to `ServletContext` as `WebApp` is a child of `ServletContext`.